test/nodes/test_prompt_node.py

```diff
@@ -763,6 +763,35 @@ def test_complex_pipeline_with_multiple_same_prompt_node_components_yaml(tmp_pat
     assert pipeline is not None
 
 
+class TestTokenLimit:
+    def test_hf_token_limit_warning(self, prompt_node, caplog):
+        tt = PromptTemplate(
+            name="too-long-temp",
+            prompt_text="Repeating text" * 200 + "Docs: $documents; Answer:",
+            prompt_params=["documents"],
+        )
+        with caplog.at_level(logging.WARNING):
+            _ = prompt_node.prompt(tt, documents=["Berlin is an amazing city."])
```

Comment:
Just a small side note that calling the `PromptTemplate` `prompt_template` would be more readable than `tt`.